components/policy/resources/templates/policy_definitions/BraveSoftware/BraveWalletDisabled.yaml

```diff
@@ -0,0 +1,37 @@
+caption: Disable Brave Wallet
+default: null
+desc: |-
+  Disable Brave Wallet in Brave.
+
+  Allows users to use the secure, built-in crypto wallet that supercharges the browser for Web3.
```

Comment:
cc: @Miyayes on the wording here! I grabbed it from https://brave.com/wallet/. This text will show in the Windows group policy editor. It looks like this when it's displayed
![image](https://github.com/user-attachments/assets/a0216fe5-819e-42e8-afe3-5fdc69a3a351)